picATrip/urls.py

```diff
@@ -30,7 +30,7 @@
     path('login/', auth_views.LoginView.as_view(template_name='users/login.html'), name='login'),
     path('logout/', auth_views.LogoutView.as_view(template_name='users/logout.html'), name='logout'),
     path('profile/', user_views.profile, name='profile'),
-    path('postsList/', post_views.PostListView.as_view(), name='view posts')
+    path('postList/', include('Post.urls')),
```

Comment:
Did you try include views.PostListView.as_view()
Did you think of another context in PostApp ? 
such as default /post and in tsub context post/postList

Reply:
I wanted to change the post part as little since my main focus was the commenting part, so I did the minimum change in order to have the commenting system working, maybe this should be in next PRs when the person who's responsible to the post part will take care of it, is it acceptable. And I didn't see any difference in the view after this change, not sure what the thing you suggested will do, I wanted to use the include to not have to add here some routing that is more specific and relevant to the post and using a urls.py separate file, and also to experience doing such a thing with django

Reply:
LGTMT